keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
It makes sense!

>  What I'm trying to say is that if the data blob is already in the DB, and we store the user's policies as part of the blob, we don't need to also store them in the DB again.

Yup, I got it. However, in my latest implementation, I decided to not store all the data blob in the DB anymore since I see there is no reasons to do it. What do you think?

> I think the policies must be revealed to the client, otherwise the client won't be able to parse the rest of the data blob. In a proof of absence, the policies shouldn't be included anyway since the entire blob is dropped from the proof. But if the server removes the user policies before responding to a proof of inclusion, how will the client know if the data it's receiving is the user's key or an encrypted blob? And how will the client know if it should look for a signature on a key change message?

In the case of the lookup policy, revealing the policy to the client shouldn't violate the user's privacy if the client isn't authorized to decrypt the encrypted blob. In the case of the key change policy, I do think there might be a risk to exposing the policy to a malicious client (which might attempt to change a user's key if it detects that the user allows unsigned key changes). But I think the solution to this issue might be some kind of account authentication again.

Okay, we should store the user policies as part of the blob mapped to the user's index as well. Makes sense! 
I would like to open separate issue for the key change function, though.
Just a side-note, we should de-serialize the data blob to user's key, user policies before responding (it maybe looks like `{"key": "...", "allow_public_lookup": true, "allow_unsigned_key_change": true}` instead of `{"blob": "allow_public_lookup||allow_unsigned_key_change||key"}`